src/Package/Impl/Sql/Publish/SProcGenerator.cs

```diff
@@ -67,7 +67,7 @@ internal sealed class SProcGenerator {
         private void CreateRCodeTable(SqlSProcPublishSettings settings, EnvDTE.Project targetProject, string targetFolder, EnvDTE.ProjectItem targetProjectItem) {
             var creatTableScriptFile = Path.Combine(targetFolder, CreateRCodeTableScriptName);
             using (var sw = new StreamWriter(creatTableScriptFile)) {
-                sw.WriteLine(Invariant($"CREATE TABLE {settings.TableName}"));
+                sw.WriteLine(Invariant($"CREATE TABLE [{settings.TableName}]"));
```

Comment:
As for Ms SQL specific syntax, yes the target is sproc written in R and executed in MS SQL as in https://msdn.microsoft.com/en-us/library/mt591996.aspx